quinn-proto/src/connection/mod.rs

```diff
@@ -422,6 +428,8 @@ where
                 || self.path_response.is_some()
         });
 
+        let mut was_congestion_blocked = false;
```

Comment:
Elsewhere (e.g. `handle_packet`) we use "was_*" to refer to the state prior to running some logic which may change that state, whereas this is referring to the state after we're done. Could we say `is_` here instead? Also, maybe `transport_blocked` (as opposed to application) since pacing can be responsible as well as congestion?

Reply:
Fine with removing `was_`. But I think even with pacing `congestion_blocked` is fine, because pacing is a form of congestion control